commands/addons/create.js

```diff
@@ -2,12 +2,13 @@
 
 const cli = require('heroku-cli-util')
 const co = require('co')
+let waitForAddonProvisioning = require('../../lib/addons_wait')
 
 function parseConfig (args) {
   let config = {}
   while (args.length > 0) {
     let key = args.shift()
-    if (!key.startsWith('--')) throw new Error(`Unexpected argument ${key}`)
+    if (!key.startsWith('--')) { throw new Error(`Unexpected argument ${key}`) }
```

Comment:
We don't necessarily know what add-on-specific options are allowed on create, though, since it will vary by add-on and perhaps even by plan / region. Thoughts?